packages/shared/src/components/sidebar/sections/MainSection.tsx

```diff
@@ -33,11 +36,12 @@ export const MainSection = ({
   }, [logSubscriptionEvent]);
 
   const menuItems: SidebarMenuItem[] = useMemo(() => {
+    const myFeedPath = isCustomDefaultFeed ? `${webappUrl}my-feed` : '/';
```

Comment:
But this will redirect extension users to webapp right for my feed?
In ideal world we keep them on extension for both custom feeds and my feed as we do for the other feeds for instance now (explore/bookmarks)

Reply:
from extension testing it looks like when I have custom feed and click on my feed in sidebar it does not open webapp